recipes/nionswift/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "nionswift" %}
+{% set version = "0.14.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  #url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  #sha256: 1f0aae216af499bf0c4e6e2571bd37d53ee75096866fa4266c9014d6d31fffb5
+  url: https://github.com/nion-software/nionswift/archive/{{ version }}.tar.gz
```

Comment:
We temporary don't use the source from pypi, since there are missing files in the source distribution, see https://github.com/nion-software/nionswift/pull/333.

Reply:
Several other projects have this same problem: `nionswift-instrumentation`, `nionswift-eels-analysis`, `nionswift`. I have committed fixes so that they are fixed in the next release for each.